agent_build_refactored/managed_packages/managed_packages_builders.py

```diff
@@ -327,13 +328,232 @@ def get_agent_libs_build_command_args(cls) -> List[str]:
             # fmt: on
         ]
 
-    def _build_agent_package(
-        self,
-        agent_libs_package_version: str,
-    ) -> str:
+    def _prepare_package_python_and_libraries_files(self, package_root: pl.Path):
```

Comment:
Previously we created files for dependency packages in a separate steps by using those scripts: 
`agent_build_refactored/managed_packages/steps/build_agent_libs_package_root.sh`, `agent_build_refactored/managed_packages/steps/build_python_package_root.sh`. Now we do not create separate dependency packages, but still need to prepare those files, so everything just moved from those script to this function.